scripts/tests/test-runner.js

```diff
@@ -160,6 +164,12 @@ function getCallback(test, callback) {
 // -- Build tests (each endpoint has a section) --
 const testCases = [
   function(callback) {
+    if (args.name === 'mongodb') {
```

Comment:
Is there a list somewhere to store all the supported databases? I'm assuming more will be added in the future.

Reply:
Yeah I'll keep the docs updated! This change you commented on is kinda a hack for the integration test right now - I'll fix this asap